src/SFML/Graphics/Texture.cpp

```diff
@@ -364,7 +381,8 @@ Image Texture::copyToImage() const
         // Texture is either padded or flipped, we have to use a slower algorithm
 
         // All the pixels will first be copied to a temporary array
-        std::vector<Uint8> allPixels(static_cast<std::size_t>(m_actualSize.x) * static_cast<std::size_t>(m_actualSize.y) * 4);
+        std::vector<Uint8> allPixels(
+            static_cast<std::size_t>(m_actualSize.x) * static_cast<std::size_t>(m_actualSize.y) * 4);
```

Comment:
Personally, I could live with longer max line sizes. Quite a few of these changes make it IMHO less readable

Reply:
I could live with that too. 
```diff
-    HWND     m_handle;        //!< Win32 handle of the window
-    LONG_PTR m_callback;      //!< Stores the original event callback function of the control
-    bool     m_cursorVisible; //!< Is the cursor visible or hidden?
-    HCURSOR m_lastCursor; //!< Last cursor used -- this data is not owned by the window and is required to be always valid
-    HICON   m_icon;       //!< Custom icon assigned to the window
-    bool    m_keyRepeatEnabled; //!< Automatic key-repeat state for keydown events
-    Vector2u m_lastSize;        //!< The last handled size of the window
-    bool     m_resizing;        //!< Is the window being resized?
-    Uint16 m_surrogate; //!< First half of the surrogate pair, in case we're receiving a Unicode character in two events
-    bool   m_mouseInside;   //!< Mouse is inside the window?
-    bool   m_fullscreen;    //!< Is the window fullscreen?
-    bool   m_cursorGrabbed; //!< Is the mouse cursor trapped?
+    HWND     m_handle;           //!< Win32 handle of the window
+    LONG_PTR m_callback;         //!< Stores the original event callback function of the control
+    bool     m_cursorVisible;    //!< Is the cursor visible or hidden?
+    HCURSOR  m_lastCursor;       //!< Last cursor used -- this data is not owned by the window and is required to be always valid
+    HICON    m_icon;             //!< Custom icon assigned to the window
+    bool     m_keyRepeatEnabled; //!< Automatic key-repeat state for keydown events
+    Vector2u m_lastSize;         //!< The last handled size of the window
+    bool     m_resizing;         //!< Is the window being resized?
+    Uint16   m_surrogate;        //!< First half of the surrogate pair, in case we're receiving a Unicode character in two events
+    bool     m_mouseInside;      //!< Mouse is inside the window?
+    bool     m_fullscreen;       //!< Is the window fullscreen?
+    bool     m_cursorGrabbed;    //!< Is the mouse cursor trapped?
```
These are the kinds of changes we get when increasing the column width.

Reply:
Absolutely agree. We live in a time where screens tend to be super wide 🙂

Reply:
Just pick whatever bigger width you want and I'll make the change.

Reply:
Let's just go with what we have right now, maybe sometime later, someone is bothered enough to define a wider width